extensions/wikia/PageHeader/Language.class.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Wikia\PageHeader;
+
+use \RequestContext;
+use \WikiaApp;
+
+class Language {
+
+	public $currentLangName;
+	public $languages;
+	public $languageUrls;
+	private $langCode;
+	private $title;
+
+	/**
+	 * Language constructor.
+	 *
+	 * @param WikiaApp $app
+	 */
+	public function __construct( WikiaApp $app ) {
+		$this->langCode = RequestContext::getMain()->getLanguage()->getCode();
+
+		$this->currentLangName = \Language::getLanguageName( $this->langCode );
```

Comment:
Language code is used only here, there is no need to set a class property. A local variable would be enough, or even no variable.